Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs

```diff
@@ -85,7 +86,7 @@ public void OnLayout(bool changed, int left, int top, int right, int bottom)
 				return;
 
 			AppCompatButton view = View;
-			if (view == null || view.Layout == null)
+			if (view == null)
```

Comment:
@mattleibow do you by chance remember why this null check is here? I found that with it there would sometimes be cases where the layout would be null and then the padding code below would never run. So Just removed it and that fixed my scenario but I'm not sure if I broke a different one